### PR TITLE
Consistenly refer to account identifier as accountId

### PR DESCRIPTION
**Pull request by gjvis**
Avoids confusion around "id", particularly in mustache templates

### CODE REVIEW

File: routes/dashboard.js [hunk 1]
```diff
@@ -5,15 +5,15 @@ const Jobs = require('../models/jobs-model');
...
 });
 
-router.get('/:id', (req, res, next) => {
+router.get('/:accountId', (req, res, next) => {
   Jobs
-    .findAllByAccountId(req.params.id)
+    .findAllByAccountId(req.params.accountId)
     .then((jobs) => res.render('index',
-      { title: 'Dashboard', id: req.params.id, jobs: jobs.toJSON() }))
+      { title: 'Dashboard', accountId: req.params.accountId, jobs: jobs.toJSON() }))
```
Comment (lukasz-kaniowski) [inline routes/dashboard.js]: to avoid duplication `const accountId = req.query.id ` ?

Comment (gjvis) [inline routes/dashboard.js]: yeah, i considered doing that to reduce the size of the closed-over scope. will implement

File: routes/jobs.js [hunk 2]
```diff
@@ -3,18 +3,22 @@ const router = new express.Router({ mergeParams: true });
...
+      { errors: req.validationErrors() },
+      { accountId: req.params.accountId },
+      req.body
+    );
     return res.render('jobs-new', body);
   }
-  return new Jobs(Object.assign(req.body, { accountId: req.params.id })).save()
-    .then(() => res.redirect(`/${req.params.id}`))
+  return new Jobs(Object.assign(req.body, { accountId: req.params.accountId })).save()
+    .then(() => res.redirect(`/${req.params.accountId}`))
```
Comment (lukasz-kaniowski) [inline routes/jobs.js]: same as above => to avoid duplication `const accountId = req.params.accountId` ?

**Comment by lukasz-kaniowski** [comment]
:shipit:

**Pull request merged**